.changeset/tidy-signals-shine.md

```
@@ -0,0 +1,5 @@
+---
+'@mastra/playground-ui': patch
+---
+
+Simplify the Trace Intelligence signal settings pane and correct index controls and pending-signal elevation styling.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)